pkg/cmd/cli/cmd/process.go

```diff
@@ -89,7 +110,7 @@ Examples:
 			// If 'parameters' flag is set it does not do processing but only print
 			// the template parameters to console for inspection.
 			if cmdutil.GetFlagBool(cmd, "parameters") == true {
-				err = printer.PrintObj(templateObj, out)
+				err = printTemplateParameters(templateObj, out)
```

Comment:
@smarterclayton template parameters are not API objects AFAIK, I think I tried this before and the PrintObj require runtime.Object